start-testnet-client.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+# Copyright (c) 2018-2020 MobileCoin Inc.
+#
+# Launches a local `mobilecoind` instance that syncs the ledger from two nodes in the
+# test network and hosts a wallet service running on port 4444, then launches a local
+# `mc-testnet-client` instance that interacts with the local `mobilecoind`.
+
+set -e
+
+trap "trap - SIGTERM && kill -- -$$" SIGINT SIGTERM EXIT
+
+source "$HOME/.cargo/env"
+
+pushd "$(dirname "$0")"
+
+echo "Pulling down TestNet consensus validator signature material"
+curl -O https://enclave-distribution.test.mobilecoin.com/pool/e57b6902aee60be45b78b496c1bef781746e4389/bf7fa957a6a94acb588851bc8767eca5776c79f4fc2aa6bcb99312c3c386c/consensus-enclave.css
+
+TARGETDIR=../target/release
+
+if [[ ! -f ${TARGETDIR}/mc-testnet-client ]] || [[ ! -f ../../../target/release/mobilecoind ]]; then
```

Comment:
Whats with the `../../../`?

Reply:
I think we should always run `cargo build` even if the files exist. That ensures the correct version gets built, for example after pulling.